code/game/objects/items/weapons/RCD.dm

```diff
@@ -428,6 +428,8 @@
 	if(iswallturf(A))
 		if(istype(A, /turf/simulated/wall/r_wall) && !canRwall)
 			return FALSE
+		if(istype(A, /turf/simulated/wall/indestructible))
+			return FALSE
```

Comment:
this should probably be defines under walls_indestructible.dm, if it is not already.